pkg/cloudfabric-controller/controllerframework/controller_framework_utils.go

```diff
@@ -54,6 +53,31 @@ func SortControllerInstancesByKeyAndConvertToLocal(controllerInstanceMap map[str
 	return sortedControllerInstancesLocal
 }
 
+// evenly split 0 ~ maxInt64 interval
+func reassignControllerKeys(instances []controllerInstanceLocal) []controllerInstanceLocal {
+	if len(instances) == 0 {
+		return instances
+	}
+
+	if len(instances) == 1 {
```

Comment:
line 62-64 can be deleted. the logic in line 67-76 will handle the case that len(instances) ==0.

Reply:
Yes. And there is test cover this scenario. Removed.